MODULE.bazel

```diff
@@ -7,7 +7,7 @@ module(
 )
 
 bazel_dep(name = "rules_license", version = "0.0.3")
-bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
```

Comment:
rules_testing currently specifies skylib=1.3.0, so upgrading the version here shouldn't be necessary? In any case, either rules_testing needs to update its version, or this can remain at 1.3.

If rules_testing needs to be updated, can you file an issue describing the failure? I wouldn't be surprised if test coverage is missing.

Reply:
Changed both the MODULE.bazel and the WORKSPACE reference to 1.3.0 as that does appear to be good enough.